docs/tutorials/learn.md

```diff
@@ -569,7 +569,7 @@ This is not acceptable if you care about performance. In an imperative
 language, one would use a simple loop going over the list without any
 memory allocation. In pseudocode:
 
-```juvix
+```pascal
```

Comment:
We can try "delphi" or "Delphi" here. Then some reasonable highlighting should work.

Reply:
Done.